docker/Dockerfile.node

```diff
@@ -6,25 +6,33 @@ RUN apt-get update && apt-get install -y  --no-install-recommends \
 
 WORKDIR /build
 
-# Cache dependencies
+# Target binary name (e.g., mojave-node, mojave-sequencer, mojave-prover)
+ARG TARGET_BIN="mojave-node"
```

Comment:
since the dockerfile are seperated now, this shouldn't be needed anymore
(same in all dockerfiles)